CONTRIBUTING.md

```diff
@@ -1,4 +1,6 @@
-<img width=250px src="https://atsign.dev/assets/img/atPlatform_logo_gray.svg?sanitize=true">
+<img src="https://atsign.dev/assets/img/@dev.png?sanitize=true">
```

Comment:
This file shouldn't be changed, and the proposed change is reverting to an old version.

Reply:
Fixed